components/Tabs/src/MDCTabBar.m

```diff
@@ -83,9 +73,23 @@ @interface MDCTabBar () <MDCItemBarDelegate>
 @implementation MDCTabBar {
   /// Item bar responsible for displaying the actual tab bar content.
   MDCItemBar *_itemBar;
+
+  // Flags tracking if properties are unset and using default values.
```

Comment:
I think we can implement this using UIAppearance proxy.
Can we catch up offline and discuss?